server/lib/models/pointer.ts

```diff
@@ -85,6 +115,24 @@ const PointerModel = (sequelize, DataTypes) => {
     return pointerIds;
   };
 
+  Pointer.prototype.newDirectContainedPointerIds = async function(pointersSoFar) {
+    const value = await this.value;
+    if (!value) {
```

Comment:
When can this happen?

Reply:
I don't know. There are a lot of guards like this throughout the codebase where it's unclear how the situation guarded against can arise. I think it would take quite a bit of detective work to figure out if they're still needed. Seems faster to just minimally modify the `directContainedPointerIds` method.